less/components/expander.less

```diff
@@ -21,6 +21,9 @@ html.no-js {
     font-weight: bold;
     text-transform: uppercase;
     padding: 2rem;
+    margin: 0;
+    font-size: inherit;
```

Comment:
Oh, I think it is supposed to override it. I just added the rules necessary for the `h4` to not have any visual difference from the `div`... I believe that `font-size: inherit` is needed to make the headings look like they used to on the web literacy page, while the overridden `font-size: 1.5rem` is needed to make them look like they used to on the clubs toolkit page?